sklearn/datasets/_arff_parser.py

```diff
@@ -371,11 +371,35 @@ def _pandas_arff_parser(
         names=[name for name in openml_columns_info],
         dtype=dtypes,
     )
-
     columns_to_select = feature_names_to_select + target_names_to_select
     columns_to_keep = [col for col in frame.columns if col in columns_to_select]
     frame = frame[columns_to_keep]
 
+    # strip quotes to be consistent with LIAC ARFF
+    re_start_end_quotes = ['^["](.*["])$', "^['](.*['])$"]
```

Comment:
```python
pattern = r"^(?P<quotationsymbol>[\"'])(?P<contents>.*)(?P=quotationsymbol)$"
```

then you can use the following to access the contents in case of a match:

```python
    def strip_quotes(s):
        return s.group("contents")
```

Reply:
More info in: https://www.regular-expressions.info/named.html